rebar.config

```diff
@@ -14,13 +12,22 @@
             warn_unused_import,
             warnings_as_errors]}.
```

Comment:
Just so I am clear, you want me to reinstate `debug_info` in the `-ifdef(TEST)` blocks of the Erlang modules, is that right?